.travis.yml

```diff
@@ -73,10 +71,8 @@ jobs:
         - go get -u golang.org/x/lint/golint
       name: lint
       script:
-        - bin/lint
-        - bin/vet
-        - bin/staticcheck
```

Comment:
This looks like staticcheck has been removed?

Reply:
It has been moved as part of the lint process (as it was in quarks-jobs) https://github.com/cloudfoundry-incubator/quarks-utils/blob/master/bin/lint#L5.  I would go over to drop the separate job in the pipeline in case we merge this

Reply:
Not sure this has the same fail behavior as lint. Go lint is not meant for failing PRs (https://github.com/golang/lint#purpose). 

Also I suspect vet and lint need to download modules to work, but staticcheck does not. On concourse we would do less work if vet and lint were running in one task.

Reply:
:+1:  I can amend that and run staticcheck only in travis here as I've left the target in the makefile, but the fix can go in another PR for quarks-utils as the common scripts are already there

Reply:
I've just checked, we were already making it fail with `bin/lint`, should we change behavior here ? I don't have strong feelings on this

Reply:
I remember arguing about that before, but let's just keep it for now